GDJS/GDJS/Events/CodeGeneration/MetadataDeclarationHelper.cpp

```diff
@@ -1545,7 +1548,58 @@ gd::ObjectMetadata &MetadataDeclarationHelper::GenerateObjectMetadata(
       instructionOrExpression.SetPrivate();
   }
 
+  UpdateCustomObjectDefaultBehaviors(project, objectMetadata);
```

Comment:
Isn't this operation risky? This will "refactor" the project, is there a reason why it's necessary to do it now?

Reply:
It's useful in the following cases:
- a user update an extension that declare new capa
- a user change the object capa in the extension editor